ide/db/src/org/netbeans/modules/db/explorer/dlg/Bundle.properties

```diff
@@ -300,7 +300,7 @@ ConnectionProgress_Established=Connection established.
 ConnectionProgress_Failed=Connection failed.
 ViewProcedureDialog.lName.text=&Name:
 ViewProcedureDialog.lParams.text=&Parameters:
-ViewProcedureDialog.lBody.text=&Source:
+ViewProcedureDialog.lBody.text=&Body:
```

Comment:
Any reason for changing this? I think Source matches as well as Body - do we need the bikeshedding?

Reply:
@sbrbot you are bikeshedding. Wether you call it Procedures or Routines does not matter. It does not help your cause. So please remove it. You still have not updated the PR (not even the things you aggreed to change).